src/Compilers/CSharp/Test/Semantic/Semantics/QueryTests.cs

```diff
@@ -4442,5 +4442,93 @@ class Program
                 Diagnostic(ErrorCode.ERR_QueryNoProvider, "T").WithArguments("T", "Select").WithLocation(17, 32)
                 );
         }
+
+        [Fact, WorkItem(50316, "https://github.com/dotnet/roslyn/issues/50316")]
+        public void SetOnlyProperty()
```

Comment:
This has been crashing for who knows how long. I guess no one has attempted to query syntax a set-only property before and bothered to tell us that it crashed the compiler for them.